### PR TITLE
Blog: add security release blog posts

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://github.com/nodejs-private/security-release/pull/21

### CODE REVIEW

**Review by richardlau — APPROVED** [review]

File: pages/en/blog/vulnerability/april-2024-security-releases-2.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
-date: 2024-04-04T03:00:00.000Z
+date: 2024-04-10T00:00:00.000Z
 category: vulnerability
-title: Tuesday, April 9, 2024 Security Releases
-slug: april-2024-security-releases-2
-layout: blog-post
-author: The Node.js Project
+title:  Wednesday, April 10, 2024 Security Releases
+slug: april-2024-security-releases
+layout: blog-post.hbs
```
Comment (AugustinMauroy) [inline pages/en/blog/vulnerability/april-2024-security-releases-2.md]:
```suggestion
layout: blog-post
```

**Review by marco-ippolito — APPROVED** [review]

File: pages/en/blog/vulnerability/april-2024-security-releases-2.md [hunk 2]
```diff
@@ -1,10 +1,27 @@
 ---
-date: 2024-04-04T03:00:00.000Z
+date: 2024-04-10T00:00:00.000Z
 category: vulnerability
-title: Tuesday, April 9, 2024 Security Releases
-slug: april-2024-security-releases-2
-layout: blog-post
-author: The Node.js Project
+title:  Wednesday, April 10, 2024 Security Releases
+slug: april-2024-security-releases
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/april-2024-security-releases-2.md]:
```suggestion
slug: april-2024-security-releases-2
```

File: pages/en/blog/vulnerability/april-2024-security-releases-2.md [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+
+## Command injection via args parameter of child_process.spawn without shell option enabled on Windows (CVE-2024-27980) - (HIGH)
+
+Due to the improper handling of batch files in child_process.spawn / child_process.spawnSync, a malicious command line argument can inject arbitrary commands and achieve code execution even if the shell option is not enabled.
+
+Impact:
+
+- This vulnerability affects all users in active release lines: 18.x, 20.x, 21.x
+
+Thank you, to ryotak for reporting this vulnerabilityand thank you Ben Noordhuis for fixing it.
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/april-2024-security-releases-2.md]:
```suggestion
Thank you, to ryotak for reporting this vulnerability and thank you Ben Noordhuis for fixing it.
```

**Comment by RafaelGSS** [comment]
@marco-ippolito see the reviews for the automation, we need to fix that

File: pages/en/blog/vulnerability/april-2024-security-releases-2.md [hunk 4]
```diff
@@ -1,10 +1,27 @@
 ---
-date: 2024-04-04T03:00:00.000Z
+date: 2024-04-10T00:00:00.000Z
 category: vulnerability
-title: Tuesday, April 9, 2024 Security Releases
+title:  Wednesday, April 10, 2024 Security Releases
```
Comment (MattIPv4) [inline pages/en/blog/vulnerability/april-2024-security-releases-2.md]:
```suggestion
title: Wednesday, April 10, 2024 Security Releases
```

**Comment by marco-ippolito** [comment]
> @marco-ippolito see the reviews for the automation, we need to fix that

Got it

**Review by UlisesGascon — APPROVED** [review]

**Review by AugustinMauroy — APPROVED** [review]

**Pull request merged**